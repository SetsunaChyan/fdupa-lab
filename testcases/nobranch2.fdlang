function main(){
x = 1;
y = 2;

tmp = x + y;
x = tmp - x;
y = tmp - x;

check_interval(x, 1, 1);
check_interval(y, 1, 1);
check_interval(x, 2, 2);
check_interval(y, 2, 2);
check_interval(x, 3, 3);
check_interval(y, 3, 3);

tmp = x + y;
x = tmp - x;
y = tmp - x;

check_interval(x, 1, 1);
check_interval(y, 1, 1);
check_interval(x, 2, 2);
check_interval(y, 2, 2);
check_interval(x, 3, 3);
check_interval(y, 3, 3);
}