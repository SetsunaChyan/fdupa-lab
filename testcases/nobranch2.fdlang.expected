Line 9:  NO; x in [2, 2]
Line 10: YES; y in [1, 1]
Line 11: YES; x in [2, 2]
Line 12:  NO; y in [1, 1]
Line 13:  NO; x in [2, 2]
Line 14:  NO; y in [1, 1]
Line 20: YES; x in [1, 1]
Line 21:  NO; y in [2, 2]
Line 22:  NO; x in [1, 1]
Line 23: YES; y in [2, 2]
Line 24:  NO; x in [1, 1]
Line 25:  NO; y in [2, 2]
